### DISCUSSION TITLE
Where to find executables

### DISCUSSION THREAD

**Discussion by CoordinatePlanes** [streamlink/streamlink-twitch-gui#983] [Q&A] [answered]
Installed streamlink GUI and streamlink as instructed but honestly i can't find anything that streamlink GUI will accept as acceptable for watching streams.

My streamlink looks like this but nothing works. 

![image](https://github.com/streamlink/streamlink-twitch-gui/assets/42883401/811b193c-4d52-40a7-952b-86cf53bd8bfe)

**Comment by bastimeyer** [comment, marked as answer]
Why are you trying to set custom executable paths?

https://github.com/streamlink/streamlink-twitch-gui/wiki/Streamlink-configuration#selecting-a-streaming-provider

> By default, Streamlink Twitch GUI will try to find Streamlink's executable in one of the directories listed in the system's PATH environment variable. It also includes a list of known default locations in case the file can't be resolved automatically. A custom path, relative or absolute, needs to be set if Streamlink has been installed in a different location.
>
> On Windows, Streamlink Twitch GUI will look for the streamlinkw.exe executable instead of streamlink.exe (notice the trailing w), so that no command line window will pop up when launching a stream.

If your system configuration requires setting custom executable paths, then you can find the `streamlinkw.exe` in the `bin` directory ("binaries").

Streamlink's Windows installer however always adds itself to the system's `PATH` env var, so Streamlink Twitch GUI can pick up the `streamlinkw.exe` automatically. And as a backup mechanism, Streamlink Twitch GUI will check the default path the Streamlink Windows installer chooses, so even if the path was not added to the `PATH` env var, then this should also work unless you chose a different/custom one.

**Reply by CoordinatePlanes** [reply]
Hey, selecting streamlinkw.exe in the bin directory worked. Thank you for your help. Streamlink didn't find the .exe when i installed it so that's why i was trying to find the right app to select.
